docs/UserGuide.md

```diff
@@ -145,6 +141,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [g/GENDER] [d/DOB]
   The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the contact list index.
 - At least one of the optional fields must be provided.
 - Existing values will be updated to the input values.
+- Date format accepted is: dd/mm/yyyy.
 - When editing tags, the existing tags of the person will be removed (i.e. adding of tags is not cumulative).
 - You can remove all the person’s tags by typing t/ without specifying any tags after it.
```

Comment:
can remove this too

Reply:
actually as qiaoran did the refactoring for edit command i left this here for her to remove, but ill remove it anyways